### PR TITLE
Baseline for Wasserstein figure

### PR DESCRIPTION
**Pull request by jclagarde**
Run lineage using 1 state in order to get baseline value

### CODE REVIEW

File: lineage/figures/figure5.py [hunk 1]
```diff
@@ -16,6 +16,11 @@
     commonAnalyze
 )
 from ..LineageTree import LineageTree
+from ..states.StateDistributionGamma import StateDistribution
+
+pi2 = [1]
+T2 = np.array([1])
+E3 = [StateDistribution(0.99, 8, 6)]
```
Comment (Farnazmdi) [inline lineage/figures/figure5.py:24]: replace them with:
`pi = np.ones(len(E3)) / len(E3)
T = np.eye(len(E3)`
and define E above them.

Comment (Farnazmdi) [reply lineage/figures/figure5.py:24]: make sure to `git pull`

**Comment by jclagarde** [comment]
I think that trying to do 1 state runs into lots of issues with the code. I tried commenting this out as well as other sections that refer to two states 

Screen Shot 2020-10-18 at 9 00 58 PM

**Comment by Farnazmdi** [comment]
I don't think it is lots of issues. You only need to add an if statement above line 136.
`if len(tHMMobj.X[0].E) == 2:`
 `    <<line 136...>>`
`else:`
`    pass`.

**Comment by Farnazmdi** [comment]
You shouldn't touch the way the population is being created, and shouldn't touch the last line for now.

**Pull request closed without merging**